doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1031,6 +1031,51 @@ we have to construct an infinite proof of equality, that is, an infinite object
 of type :g:`(EqSt s1 s2)`. We will see how to introduce infinite objects in
 Section :ref:`cofixpoint`.
 
+Caveat
+++++++
+
+The ability to define co-inductive types by constructors, hereafter called
+*positive co-inductive types*, is known to break subject reduction. It is a
+fundamental issue related to the duality of computational behavior between
+inductive and co-inductive types.
```

Comment:
Can you explain more? I thought that the problem was about the undecidability of equality over infinite objects. Are you saying that a similar dual subject reduction problem can be obtained with inductive types?